<launch>
    <rosparam file="$(find brobot_cam)/config/cam_config.yaml" />
    <node name="orientation_to_pose" pkg="brobot_cam" type="orientation_to_pose" />
    <node name="serial" pkg="rosserial_python" type="serial_node.py" args="/dev/ttyACM0" />
    <node type="rviz" name="rviz" pkg="rviz" args="-d $(find brobot_cam)/rviz/imu_pose.rviz" />
</launch>
